feign-reactor-spring-configuration/src/main/java/reactivefeign/spring/config/ReactiveFeignClientFactoryBean.java

```diff
@@ -287,6 +288,9 @@ public Object getObject() throws Exception {
 			} else {
 				url = this.url;
 			}
+			if (builder instanceof CloudReactiveFeign.Builder) {
```

Comment:
we need to add tests in order to verify that issue is fixed and won't be broken in future